CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(EXTENSION_CONTRIBUTORS "Maxime Gillot (University of Michigan), Baptiste Baq
 set(EXTENSION_DESCRIPTION "This extension will allow clinicians to perform automatic CBCT scan segmentation as well as automatic lamndmark identification in CBCT and IOS using machine learning tools.")
 set(EXTENSION_ICONURL "https://raw.githubusercontent.com/DCBIA-OrthoLab/SlicerAutomatedDentalTools/main/SlicerAutomaticTools.png")
 set(EXTENSION_SCREENSHOTURLS "https://raw.githubusercontent.com/DCBIA-OrthoLab/SlicerAutomatedDentalTools/main/ADT-exemple.png")
-set(EXTENSION_DEPENDS SlicerDentalModelSeg) # Specified as a list or "NA" if no dependencies
+set(EXTENSION_DEPENDS "SlicerDentalModelSeg;SlicerConda")# Specified as a list or "NA" if no dependencies
```

Comment:
This should be a space-separated list; for example see https://github.com/DCBIA-OrthoLab/SlicerCMF/blob/master/CMakeLists.txt

You're correct this should be the extension name, `SlicerConda`. Specifically it is the name of the relevant `s4ext` in https://github.com/Slicer/ExtensionsIndex/ used at extension installation.

Once this is updated we'll also need to update https://github.com/Slicer/ExtensionsIndex/blob/main/AutomatedDentalTools.s4ext with the same change.